mono/metadata/sgen-tarjan-bridge.c

```diff
@@ -124,7 +130,26 @@ typedef struct _ScanData {
 	unsigned obj_state : 2;
 } ScanData;
 
+#define HEAVY_REFS_MIN 2
+#define HEAVY_COMBINED_REFS_MIN 60
 
+/* Should color be made visible to client even though it has no bridges?
+ * If fanin and fanout are both greater than 2, then returning TRUE here will add one node to
+ * the final graph and reduce the total number of edges by some number. Return true if we
+ * estimate the number of reduced edges to be enough to justify the new node.
+ * Notice fanout is capped at INCOMING_COLORS_MAX so we can't know its exact true value.
+ */
+static inline gboolean bridgeless_color_is_heavy (ColorData *data) {
+   int fanin = data->incoming_colors;
```

Comment:
Indentation is inconsistent within this patch. Here, you're using 2 space indents. In `color_merge_array_empty()`, you're using tabs.

I don't know if this file should be using tabs or spaces, but presumably the patch itself could be consistent?